lib/cog/adapters/hipchat/api.ex

```diff
@@ -42,8 +42,8 @@ defmodule Cog.Adapters.HipChat.API do
     GenServer.call(__MODULE__, {:lookup_user, id: user_id})
   end
 
-  def lookup_user(name: username) do
-    GenServer.call(__MODULE__, {:lookup_user, id: username})
+  def lookup_user(handle: username) do
```

Comment:
We should document what keys we're expecting in the adapter behavior, with types preferably so Dialyzer can help us catch this kind of stuff earlier.